lib/engine/src/trap/frame_info.rs

```diff
@@ -227,18 +227,15 @@ impl Drop for GlobalFrameInfoRegistration {
 /// dropped, will be used to unregister all name information from this map.
 pub fn register(
     module: Arc<ModuleInfo>,
-    finished_functions: &BoxedSlice<LocalFunctionIndex, FunctionBodyPtr>,
+    finished_functions: &BoxedSlice<LocalFunctionIndex, (FunctionBodyPtr, usize)>,
```

Comment:
Can we have a struct that owns both the pointer and the length? (rather than an unnamed tuple)

Reply:
Done